docs/faq.rst

```diff
@@ -26,12 +26,15 @@ or deleting archives, which may make *simultaneous* backups fail.
 Can I copy or synchronize my repo to another location?
 ------------------------------------------------------
 
-Yes, you could just copy all the files. Make sure you do that while no
-backup is running. If you copy a repository while a backup is running, 
-the lock held will be present in the copy. Thus, before using borg on the copy 
-from a different host, you need to use :ref:`break-lock` on the copied 
-repository, because Borg is cautious and does not automatically remove 
-stale locks made by a different host.
+Yes, you could just copy all the files. But you generally do not want
+to do that.
```

Comment:
well, that is maybe a bit too restrictive.

quite some people e.g. use rclone to have a copy of their local repo in the cloud, just in case their house burns down or so.

Reply:
how should i phrase this instead?

Reply:
we should mention `borg with-lock ...` here. it will also produce a lock, but it will use the lock so not other borg updates the repo while it is copied / synchronized.

the problem is not making the copy, but using borg to access more than 1 of the set of original and cloned repos (includes: just reading, updating independently, etc.).